### ISSUE TITLE
Disable anonymous messages and enforce login though e.g. OAuth2/OIDC

### ISSUE THREAD

**Issue by septatrix** [ClaperCo/Claper#107]
While the moderation feature allows getting rid of "troll" messages there is not always the manpower to have a separate moderator available. Enforcing that users must authenticate previously and thus have a visible clear-name would greatly simplify this.

For example during university lectures the (lone) lecturer could enforce that students must previously authenticate through their universities single-sign on.

**Comment by alxlion** [comment]
Yes this could be very useful, OIDC is already available but for the presenter.
